pkg/verify/tlog.go

```diff
@@ -78,14 +75,14 @@ func VerifyArtifactTransparencyLog(entity SignedEntity, trustedMaterial root.Tru
 		if err != nil {
 			return nil, err
 		}
+		inclusionVerified := false
```

Comment:
Minor: the reason this was scoped to `!online` before was because we want to ensure there was at least one of an inclusion promise or an inclusion proof (e.g. you couldn't cheat by having neither, and falling through to have your timestamp be listed as verified).

I think with the `continue` statements, and with the placement of `if inclusionVerified` inside the `for` loop, we don't actually need it for the `online` case? But also, it's not hurting anything.

Reply:
Yep, I think this looks cleaner without. Lemme know if I misunderstood something and removed too much.

Reply:
I miss `goto` statements. I genuinely think they're cleaner than `continue`....